tests/test_optics.py

```diff
@@ -25,26 +26,28 @@
 @timer
 def test_init():
     # make sure we can init with defaults
-    model = piff.Optical(template='des')
-    prof = model.getProfile([])
+    model = piff.Optical(template='des_simple')
+    prof = model.getProfile(zernike_coeff=[0,0,0,0,0.0],r0=0.15,L0=10.,g1=0.,g2=0.)
```

Comment:
I think this should be `model.getProfile([0,0,0,0,0,0,0.15,10,0,0])` to work with the standard calling syntax.

Or possibly `model.getProfile(model.kwargs_to_params())`.